ebl/v3/EBL_v3.0.0-Beta-2.yaml

```diff
@@ -3248,42 +3386,238 @@ components:
+          type: array
+          items:
+            $ref: '#/components/schemas/TaxLegalReference'
+        partyContactDetails:
           type: array
           minItems: 1
-          maxItems: 5
+          description: |
+            A list of contact details
+          items:
+            $ref: '#/components/schemas/PartyContactDetail'
+      required:
+        - partyName
+
+    Consignee:
+      type: object
+      title: Consignee
+      description: |
+        The party to which goods are consigned in the `Master Bill of Lading`.
+        
+        **Condition:** Mandatory for non-negotiable BL (`isToOrder=false`)
+      properties:
+        partyName:
+          type: string
+          pattern: ^\S(?:.*\S)?$
+          maxLength: 100
+          description: |
+            Name of the party.
+          example: IKEA Denmark
+        address:
+          $ref: '#/components/schemas/PartyAddress'
+        displayedAddress:
+          type: array
+          minItems: 1
+          maxItems: 999
+          description: |
+            The address of the party to be displayed on the `Transport Document`. The displayed address may be used to match the address provided in the `Letter of Credit`. For physical BL (`isElectronic=false`), it is only allowed to provide max 2 lines of 35 characters. No limit for electronic BL (`isElectronic=true`).
+            
+            **Condition:** if provided, the displayed address must be included in the Transport Document.
           items:
             type: string
             maxLength: 35
             description: |
-              A single address line to be used when a B/L needs to be printed.
+              A single address line
             example: Strawinskylaan 4117
-        isToBeNotified:
-          type: boolean
+        identifyingCodes:
+          type: array
+          items:
+            $ref: '#/components/schemas/IdentifyingCode'
+        taxLegalReferences:
           description: |
-            Used to decide whether the party will be notified of the arrival of the cargo.
-          example: true
+            A list of `Tax References` for a `Party`
+          type: array
+          items:
+            $ref: '#/components/schemas/TaxLegalReference'
+        partyContactDetails:
+          type: array
+          minItems: 1
+          description: |
+            A list of contact details
+          items:
+            $ref: '#/components/schemas/PartyContactDetail'
       required:
-        - party
-        - partyFunction
-        - isToBeNotified
+        - partyName
+
+    Endorsee:
+      type: object
+      title: Endorsee
+      description: |
+        The party to whom the title to the goods is transferred by means of endorsement.
+        
+        **Condition:** Can only be provided for negotiable BLs (`isToOrder=false`). If a negotiable BL does not have an `Endorsee`, the BL is said to be "blank endorsed". Note `Consignee` and `Endorsee` are mutually exclusive.
```

Comment:
Inverted boolean logic. This can only be provided for `isToOrder=true`.

Reply:
Don't know where my head was - it will be inverted